modules/qna/src/views/full/FormModal.jsx

```diff
@@ -15,6 +15,8 @@ const ACTIONS = {
 
 export default class FormModal extends Component {
   defaultState = {
+    deletedContentElements: [],
```

Comment:
I suggest not deleting the content elements created with the content picker even if you close the window. This is the behavior everywhere else (eg, on the flow editor, if you create an element and just close the window, it still gets created).

Reply:
@allardy Ok, i put this to help keep the contentManager clean, we have a lot of questions so making this manually can be hard, but since you asked, i will remove it